bin/copy_sphinxgallery.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Script to do a local install of sphinx-gallery
+rm -rf tmp sphinxgallery
+
+easy_install -Zeab tmp sphinx-gallery
+
+cp -vru tmp/sphinx-gallery/sphinxgallery/ .
+
+git add sphinxgallery
```

Comment:
Give that the last time I updated sphinx-gallery from PyPI I forgot to git add the new files, I do think is worth having this line. Even if it fails because the user does not use git, the local embedding is done and is one less thing to worry about.

https://github.com/scikit-learn/scikit-learn/pull/4658#discussion_r31616558